code/code/sys/configuration.h

```diff
@@ -1,7 +1,15 @@
 #pragma once
 
+#include <string>
+#include <string_view>
 #include "sstring.h"
 
+inline constexpr const char* MUD_NAME = "SneezyMUD";
+extern sstring MUD_NAME_VERS;
+
+std::string readFileContents(std::string_view);
+std::string readVersionFromFile();
```

Comment:
Short and simple enough be in main.cc.

Reply:
See above - I'll combine the functions and move `readVersionFromFile` into an anonymous namespace in `main.cc`.

Reply:
Fixed in https://github.com/sneezymud/sneezymud/pull/476/commits/17de1b40cc65e5f489f434878ab0267af991d881